featurebyte/feast/utils/on_demand_view.py

```diff
@@ -97,9 +97,9 @@ def create(
                 sources.append(fv_source)
                 if fv_source.ttl is not None:
                     if ttl_seconds is None:
-                        ttl_seconds = fv_source.ttl.seconds
-                    elif fv_source.ttl.seconds < ttl_seconds:
-                        ttl_seconds = fv_source.ttl.seconds
+                        ttl_seconds = fv_source.ttl.total_seconds()
+                    elif fv_source.ttl.total_seconds() < ttl_seconds:
+                        ttl_seconds = fv_source.ttl.total_seconds()
```

Comment:
Curious what was the effect of setting this incorrectly?

Reply:
It only takes the second component of the ttl. For example, when the ttl is `1d`, it treats it as non-ttl. If it is non-ttl, it will not include point in time request column.